pkg/clients/aws/route53.go

```diff
@@ -613,7 +613,7 @@ func getSTSCredentials(reqLogger logr.Logger, client *sts.STS, roleArn string, e
 			break
 		}
 		if i == (assumeRolePollingRetries - 1) {
-			return nil, fmt.Errorf("timed out while assuming role %s", roleArn)
+			return nil, fmt.Errorf("timed out assuming the role, error: %v", err)
```

Comment:
I think we ought to include the following details:
- Certman tried and failed multiple times to assumeRole, and now we're out of retries
- What errors certman received when attempting to assumeRole. We could just return the last error encountered, like you're doing now, or we could just log them as we go

Very rough & untested code that would satisfy these criteria:
```suggestion
                assumeRoleOutput, err = client.AssumeRole(&assumeRoleInput)
		if err == nil {
		        reqLogger.Errorf("failed to assumeRole (attempt %d out of %d): %v", i, assumeRolePollingRetries, err)
			break
		}
		if i == (assumeRolePollingRetries - 1)  {
			return nil, fmt.Errorf("failed to assumeRole, no retries left: %v", err)
		}
```